getting_started/example.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit (conda)",
+   "metadata": {
+    "interpreter": {
+     "hash": "1069a05e44c2652dfca512fe2a9572bc78d7076fd0ad2ea71f27c4bc70a0e70f"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "c:\\Users\\kuhnie\\code-chicago\\gcfd\\code-for-chicago-gcfd\n"
```

Comment:
Does this matter? Looks like it's a local path on your machine

Reply:
No, jupyter notebooks store that data. There might be a way to clear that